kroxylicious-filter-test-support/src/main/java/io/kroxylicious/test/condition/ApiMessageCondition.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.test.condition;
```

Comment:
Wondering if kafka conditions should belong in kafka package. i.e. `io.kroxylicious.test.condition.kafka`

Reply:
yeah thats probably a wise move.